repobee_sanitizer/_sanitize_repo.py

```diff
@@ -56,6 +59,13 @@ def _sanitize_repo(
                 args.repo_root, file_relpaths, args.target_branch
             )
 
+        if args.force:
+            return plug.Result(
+                name="sanitize-repo",
+                msg="Sanitize-repo was forced",
+                status=plug.Status.SUCCESS,
+            )
+
```

Comment:
I don't think this adds anything. The user will be aware that they forced the update. At most, this should be a log message.

```suggestion
```

Reply:
It _is_ odd that `sanitize-repo` does not return a result if it succeeds, so that should probably be fixed, but this is not the place.